Kconfig.zephyr

```diff
@@ -360,9 +360,14 @@ config NO_OPTIMIZATIONS
 	help
 	  Compiler optimizations will be set to -O0 independently of other
 	  options.
-
 endchoice
 
+config ENABLE_WARNING_AS_ERRORS
+	bool "Stop building if warnings occures"
+	default n
```

Comment:
You can drop `default n` - it's 'n' by default.

Reply:
Done